docs/en/siem/siem-ui.asciidoc

```diff
@@ -8,7 +8,31 @@ environment, and you can use the interactive UI to drill down into areas of
 interest.
 
 The *{kibana-ref}/kuery-query.html[{kib} Query Language (KQL)]* bar is available
-throughout the {siem-app} for searching and filtering.
+throughout the {siem-app} for searching and filtering. You can also click and
+swipe histograms, which updates the global time filter.
```

Comment:
> You can also click and swipe histograms, which updates the global time filter.

Would you be willing to consider describing this behavior ("brush selection" is the obscure technical term for it), like the following suggestion?

When histograms contain peaks or a time range containing interesting data, click on the chart, and while holding down the mouse or trackpad button, drag over the time range to select it, which updates the global time filter.

<img width="844" alt="brush-selection-still" src="https://user-images.githubusercontent.com/4459398/81319154-7c90cc80-904c-11ea-885f-914d4d6715fb.png">

Consider including a still screenshot like the one above to illustrate the selection process.

Do we support animated gifs in docs? If so, perhaps we could include something like the following?

![brush-selection-animated](https://user-images.githubusercontent.com/4459398/81319088-6256ee80-904c-11ea-9871-64050893ea2b.gif)

Reply:
Updated the UI text to describe what to do instead of how to do it, and the screenshot. I think it's ok now.
Technically, we can add gif animations. I need to check with other writers if it's ok to do so. Personally, I find them a bit distracting in docs.